docs/user/reference/general/register-custom-pings.md

```diff
@@ -0,0 +1,75 @@
+# Registering custom pings
+
+After defining [custom pings](../../user/pings/custom.md) The Glean SDK build generates code from `pings.yaml` in a `Pings` object, which must be instantiated so Glean can send pings by name.
+
+## Loading custom ping metadata into your application or library
+
+{{#include ../../../shared/tab_header.md}}
```

Comment:
Tabs missing :) Note that you can copy paste from [here](http://localhost:3000/appendix/contribution-guidelines.html#tabs).